packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -231,6 +234,22 @@ export async function finalizeTransformByQ(status: string) {
     }
 }
 
+export async function parseBuildFile() {
+    const absolutePaths = ['Users/', 'System/', 'Volumes/', 'C:', 'D:']
```

Comment:
is it possible that these could also exist in pom.xml as lowercase? e.g. vscode on windows and node normalize drive paths to lowercase sometimes.